frontend/pages/features/index.js

```diff
@@ -0,0 +1,157 @@
+import React, { useContext } from "react";
+import { Container } from "@chakra-ui/react";
+import RouteButton from "../../src/components/RouteButton";
+import mixpanel from "mixpanel-browser";
+import { getLayout, getLayoutProps } from "../../src/layouts/WideInfoPage";
+import { AWS_ASSETS_PATH } from "../../src/core/constants";
+import { MIXPANEL_EVENTS } from "../../src/core/providers/AnalyticsProvider/constants";
+import FeatureCard from "../../src/components/FeatureCard";
+import useRouter from "../../src/core/hooks/useRouter";
+import UIContext from "../../src/core/providers/UIProvider/context";
+
+const assets = {
+  cryptoTraders: `${AWS_ASSETS_PATH}/crypto+traders.png`,
+  lender: `${AWS_ASSETS_PATH}/lender.png`,
+  DAO: `${AWS_ASSETS_PATH}/DAO .png`,
+  NFT: `${AWS_ASSETS_PATH}/NFT.png`,
+};
+
+const Features = () => {
+  const router = useRouter();
+  const ui = useContext(UIContext);
+
+  const mixpanelReport = function (name, section) {
+    const _report = function () {
+      if (mixpanel.get_distinct_id()) {
+        mixpanel.track(`${MIXPANEL_EVENTS.BUTTON_CLICKED}`, {
+          full_url: router.nextRouter.asPath,
+          buttonName: name,
+          page: `features`,
+          section: section,
+        });
+      }
+    };
+    return _report;
+  };
+
+  return (
+    <Container id="container" maxW="container.xl">
+      {!ui.isMobileView && (
+        <RouteButton
+          variant="orangeAndBlue"
+          onClick={mixpanelReport("Learn More", "main")}
+          href={"/discordleed"}
+          isExternal
+          minW={["150", "150", "150", "200px", "300px", "300px"]}
+          fontSize={["md", "lg", "xl", "2xl", "3xl", "3xl"]}
+          position="absolute"
+          bottom="10px"
+          right="5px"
+        >
+          Learn More
+        </RouteButton>
+      )}
+      <FeatureCard
+        id="airdrops"
+        headingText="Airdrops"
+        image={assets["lender"]}
+        cardOrder={1}
+        isMobile={ui.isMobileView}
+        clickEvent={mixpanelReport("Learn More", "airdrops")}
+      >
+        <>
+          Use Moonstream to distribute ERC20 tokens, NFTs, items, or
+          achievements to your community. All you have to do is upload a
+          spreadsheet listing the amount that each community member should
+          receive.
+          <br />
+          <br />
+          Our smart contracts and APIs handle the rest. Integrate your frontends
+          or game clients with our APIs for full control over the claim
+          experience. Use Moonstream-backed leaderboards to automatically reward
+          players for their on-chain activity.
+          <br />
+          <br />
+          Gaming projects have used Moonstream to drop over $80,000,000 worth of
+          tokens and items to date.
+        </>
+      </FeatureCard>
+      <FeatureCard
+        id="minigames"
+        headingText="Minigames"
```

Comment:
For SEO reasons we need to start using `<h1> <h2> <h3>` props in our marketing pages. 
TO do that set `<Heading as="h1" >` and in case of FeatureCard - add ability to pass level property to set up heading levels

Reply:
some reading https://clictadigital.com/how-to-use-h1-h2-and-h3-header-tags-for-seo-effectively/